src/app/common/stores/applications/Distribution/Wallet.store.ts

```diff
@@ -127,8 +199,15 @@ class WalletStore extends BaseStore implements Wallet {
   async connect() {
     try {
       console.debug('Connecting wallet..');
-      await this.onboardSdk.walletSelect();
-      await this.onboardSdk.walletCheck();
+      const result = await this.onboardSdk.connectWallet();
+      if (result?.length > 0) {
+        const networkId = result[0].chains[0].id.replace('0x', '');
```

Comment:
You need to convert hex to int here, simply removing 0x will break holesky because it have integer 17000 network id